examples/close_cookies/close_cookies.py

```diff
@@ -0,0 +1,45 @@
+"""This example demonstrates how to close popup windows (like promotion form) with AgentQL."""
+
+import time
+
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# Set the URL to the desired website
+URL = "https://www.amazon.com/"
+
+QUERY = """
+{
+    cookies_form {
+        reject_btn
+    }
+}
+"""
+
+
+def main():
+    with sync_playwright() as playwright:
+        browser = playwright.chromium.launch(headless=False)
+
+        # Create a new AgentQL page instance in the browser for web interactions
+        page: Page = browser.new_page()
+
+        page.goto(URL)
+
+        # Use query_elements() method to fetch the cookies dialog button from the page
+        response = page.query_elements(QUERY)
+
+        # Check if there is a cookie-rejection button on the page
+        if response.cookies_form.reject_btn != None:
```

Comment:
If this also will show a warning, then it's better to save it in local var

```suggestion
        if response.cookies_form.reject_btn is not None:
```